plugin.json

```diff
@@ -23,6 +23,20 @@
         "help_text": "When false, users cannot attach and create Jira issues in Mattermost. Does not affect Jira webhook notifications. After changing this setting to false, disable, then re-enable this plugin in **System Console > Plugins > Plugin Management** to reset the plugin state for all users. \n \n When true, install this plugin to your Jira instance with '/jira install’ to allow users to create and manage issues across Mattermost channels. See [documentation](https://about.mattermost.com/default-jira-plugin-link-application) to learn more.",
         "default": true
       },
+      {
+        "key": "WebhookMaxProcsPerServer",
+        "display_name": "Maximum # of Webhook Processors Per Server",
+        "type": "text",
+        "help_text": "The number of simultaneous go routines that will process webhooks. This is needed because to process a webhook the Jira plugin may have to request information from the Jira instance (such as permissions), and we may not want to overload Jira. \n \n Changes to this setting will take effect when the plugin is restarted.",
+        "default": "20"
+      },
+      {
+        "key": "WebhookBufferSize",
+        "display_name": "Size of the Webhook Processing Buffer",
+        "type": "text",
+        "help_text": "The buffer size for webhook events to be processed by the webhook processors. If the buffer is full, the Jira plugin will ignore new webhook events and return a `503 Service Unavailable` response to the Jira instance that sent the webhook. This can be a large value, since each webhook event will take between 5k and 50k of memory (on average) while it's waiting to be processed. \n \n Changes to this setting will take effect when the plugin is restarted.",
```

Comment:
```suggestion
        "help_text": "The buffer size for webhook events to be processed by the webhook processors. If the buffer is full, the Jira plugin will ignore new webhook events and return a `503 Service Unavailable` response to the Jira instance that sent the webhook. This can be a large value, since each webhook event will take between 5k and 50k of memory (on average) while it's waiting to be processed. \n \n Changes to this setting will take effect when the plugin is restarted.  Default value is 10000",
```

Reply:
Since we are adding these to help avoid troubleshooting problems - the user should be aware if the limits have been changed from their defaults easily.  (That is my reasoning behind adding the value to the description 1/5)